Gems/ROS2/Code/Include/ROS2/ROS2Bus.h

```diff
@@ -50,6 +51,11 @@ namespace ROS2
         //! @note Transforms are already published by each ROS2FrameComponent.
         //! Use this function directly only when default behavior of ROS2FrameComponent is not sufficient.
         virtual void BroadcastTransform(const geometry_msgs::msg::TransformStamped& t, bool isDynamic) const = 0;
+
+        //! Obtains a simulation clock that is used across simulation.
+        //! returns constant reference to currently running clock.
+        virtual const SimulationClock& GetSimulationClock() const = 0;
+
```

Comment:
Very nice! Nit: remove extra space
Less nit: use @return (not "returns") so that doxygen properly digest it